distribution/docs/src/main/resources/content/_running/starting-intro.adoc

```diff
@@ -168,7 +168,7 @@ Change:
 #RUN_AS_USER=
 ----
 +
-to:
```

Comment:
This change was made because the field below looked like a straight 'copy-paste' change, but the user needs to change it.